app/controllers/questions/at_capacity_controller.rb

```diff
@@ -11,13 +17,5 @@ def update
       current_intake.update(continued_at_capacity: true)
```

Comment:
i dont really think that this is relevant code, because we no longer invite people to continue once we're at capacity -- we redirect them to tax slayer.